### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by teckim**
## In this pull request:

- Added user interaction test.
- Added rendering test.
- Added unit tests.

### CODE REVIEW

File: src/__tests__/calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  });
+
+  test('click', () => {
+    render(<Calculator />);
+    userEvent.click(screen.getByText('2'));
+    userEvent.click(screen.getByText('+'));
+    userEvent.click(screen.getByText('3'));
+    userEvent.click(screen.getByText('='));
+    const { value } = screen.getByTestId('testingInput');
+    expect(value).toBe('5');
```
Comment (youmari) [inline src/__tests__/calculator.test.js]: - [ ] You should test all the components as in the requirements I encourage you to do that please add tests for the navbar,home  and quote components

File: src/__tests__/operate.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import operate from '../logic/operate';
+
+describe('operate unit', () => {
+  const result = operate('1', '3', '+');
+
+  test('1 + 3 should be equale to 4', () => {
+    expect(result).toBe('4');
+  });
```
Comment (youmari) [inline src/__tests__/operate.test.js]: - [ ] Please make sure to test all the operations (-, +, /, *) and the other cases when you the user want to reset the calculator

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @teckim  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Tests passing 
✔️ No linter error 
✔️ Great title and short summary 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @teckim 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**